src/app/components/no-stores/no-stores.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+    <div class="pt-2">
+        <div class=""></div>
+        <p>Oups! It seems that there are no stores available in this category.. 😞</p>
```

Comment:
when the page is loading and fetching stores based on the selected category, the component appears in the background.
It should be only visible to the user when there is no data to be displayed.